java/configuration/src/test/java/sleeper/configuration/UtilsTest.java

```diff
@@ -38,4 +39,15 @@ void shouldCombineLists() {
         assertThat(combinedList)
                 .containsExactly("test1", "test2", "test3", "test4");
     }
+
+    @Test
+    void shouldNotThrowExceptionsDuringNumericPredicateChecks() {
+        // Given/When/Then
+        assertThatCode(() -> Utils.parseAndCheckInteger("ABC", num -> num > 0))
+                .doesNotThrowAnyException();
+        assertThatCode(() -> Utils.parseAndCheckLong("ABC", num -> num > 0L))
+                .doesNotThrowAnyException();
+        assertThatCode(() -> Utils.parseAndCheckDouble("ABC", num -> num > 0D))
+                .doesNotThrowAnyException();
```

Comment:
This could assert on the output of the public methods, which would also implicitly assert that no exception is thrown. We could have a separate test method for each one.

Reply:
By assert on the output I mean eg. `assertThat(Utils.isPositiveDouble("1.23")).isTrue()`, `assertThat(Utils.isPositiveDouble("ABC")).isFalse()`.